lib/l10n/app_en.arb

```
@@ -266,5 +315,67 @@
+  "unknown": "Unknown",
+  
+  "recurring_expense": "Recurring expense",
+  "recurring_income": "Recurring income",
+  "recurrence_type": "Recurrence type",
+  "recurrence_interval": "Interval",
+  "recurrence_none": "None",
+  "recurrence_daily": "Daily",
+  "recurrence_weekly": "Weekly",
+  "recurrence_monthly": "Monthly",
+  "recurrence_yearly": "Yearly",
+  "end_date": "End date",
```

RuboCop (1.73)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)